static_sites/labelling_librarian/README.md

```diff
@@ -0,0 +1,84 @@
+# Labelling Librarian
+
+A web-based tool that helps concept developers maintain the quality and consistency of our [labelled data](https://argilla.labs.climatepolicyradar.org/).
```

Comment:
as this is a public repo, do we want the url to our argilla instance to be in here?

Reply:
IMO including the url is fine - there are lots of methods which would allow a motivated person to discover subdomains of cpr.org (and lots of those are much more straightforward than reading through all of our documentation). if/when people do visit the site, access to the tool itself is auth'd so they would have to act in a much more malicious way to gain access... 👹